src/screens/TransactionDetailsScreen.tsx

```diff
@@ -140,14 +141,24 @@ class TransactionDetailsScreen extends Component<Props, State> {
     return (
       <View style={styles.headerContainer}>
         <Image source={transaction.value < 0 ? images.bigMinus : images.bigPlus} style={styles.image} />
+        <View style={styles.walletIconContainer}>
+          <Image
+            source={transaction.value > 0 ? icons.arrowRight : icons.arrowLeft}
+            style={styles.arrowIcon}
+            resizeMode="contain"
+          />
+          <Image source={icons.wallet} style={styles.walletIcon} resizeMode="contain" />
+        </View>
         <Text style={styles.walletLabel}>{transaction.walletLabel}</Text>
         <Text style={[styles.value, { color: transaction.value < 0 ? palette.textRed : palette.textBlack }]}>
           {`${valuePreffix}${i18n.formatBalanceWithoutSuffix(
             Number(transaction.value),
             transaction.walletPreferredBalanceUnit,
           )} ${transaction.walletPreferredBalanceUnit}`}
         </Text>
+        {this.renderLabel(transaction.tx_type)}
         <Chip
+          containerStyle={styles.chipContainer}
           label={`${transaction.confirmations < 7 ? transaction.confirmations : '6'} ${
```

Comment:
alerts have got 144 blocks, and alert_recovered don't have any check the renderCofirmations function in TransactionItem and make helper function for both something like `getConfirmationsCounter(txType, confirmations) => 8/144`

Reply:
it not quite the same component or ticket

Reply:
the confirmations should be displayed as on the transaction list view. https://projects.invisionapp.com/d/main?origin=v7#/console/20268544/426588439/preview?scrollOffset=16623 check this screen, it has 10/144 confirmations which current screen cannot display as it has only 6/6

Reply:
has confirmation counting way changed recently?

Reply:
yes, here is current
NONVAULT, RECOVERY, INSTANT -> X/6
ALERT_PENDING, ALERT_DONE -> X/144
ALERT_RECOVERED -> none